Sources/AppleTools/XcTestRun.swift

```diff
@@ -66,6 +66,10 @@ public struct XcTestRun: Equatable {
     /// The module name of this test target, as specified by the target's `PRODUCT_MODULE_NAME` build setting in Xcode.
     /// `.` and `-` symbols usually are replaced with `_`, e.g. `Some.SDK` becomes `Some_SDK`.
     public let testTargetProductModuleName: String
+    
+    public let systemAttachmentLifetime: String
+    
+    public let userAttachmentLifetime: String
```

Comment:
I think it will be nicer to use `XcTestRunAttachmentLifetime` instead of `String` here

Reply:
Plist works with strings, but you can convert back and forth. I think you can get `XcTestRunAttachmentLifetime`  from a string, and if it happens to be `nil`, just throw an error. I think this will add a validation and avoid passing unsupported values.